spec/requests/users_spec.rb

```diff
@@ -3,8 +3,23 @@
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/users/index'
+      get '/users'
       expect(response).to have_http_status(:success)
     end
+
+    it 'displays the intended template' do
+      get '/users'
+      expect(response).to render_template('index')
+    end
+
+    it 'returns the http sucess' do
+      get '/users/:id'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'displays the intended template' do
+      get '/users/:id'
+      expect(response).to render_template('show')
+    end
```

Comment:
- [ ] Kindly include tests for the `User#show` action that represents If the response body includes correct placeholder text. as per this [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.).

Reply:
Hello @gemmen29 ;

I appreciate your review on this projet. however the issues stated i think I had already implemented them. There are different `get` for each two tests (`get '/users/:id'` for `user#show` which uses `show.html.erb` as the template for verifying both placeholder and template itself, `get '/users'` for `user#index` which uses `index.html.erb as template`, also `get '/users/:user_id/posts/'` for `post#index` which uses a different `index.html.erb`  finally `get '/users/:user_id/posts/:id'` which uses a different `show.html.erb` as it acan be seen in the `routes.rb` file ). Check it out please. thanks!

Reply:
Hi @Christianib003 You should implement 3 tests for each action in both user and post controller as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.). so the total tests for each one of them will be 6 tests per controller and 12 tests total.

Reply:
@gemmen29 

The way I did Is that the test for template received both the template and the placeholder text in form of a file which either index or show and one test actually carries out two tasks. Which reduces the number of tests

Reply:
@Christianib003 
I see you already did the status code test for the `User#index` and `Post#index` actions and I edited my previous comments but you still didn't do the placeholder text test for any action. The placeholder test is to check if the view includes a specific text.